.coderabbit.yaml

```
@@ -0,0 +1,32 @@
+    enabled: true
+  gitleaks:
+    enabled: true
+
+# Code generation preferences
+code_generation:
+  docstrings:
+    language: en-US
+    format: jsdoc
+  unit_tests:
+    enabled: true
+    framework: bun_test
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)